larq_compute_engine/mlir/tests/fuse_padding.mlir

```diff
@@ -0,0 +1,54 @@
+// RUN: lce-tf-opt %s -tfl-fuse-padding -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL: @fuse_pad_into_conv_valid
+func @fuse_pad_into_conv_valid(%arg0: tensor<1x64x64x8xf32>) -> tensor<1x64x64x16xf32> {
+  %cst0 = constant dense<[[0, 0], [1, 1], [1, 1], [0, 0]]> : tensor<4x2xi32>
+  %cst1 = constant dense<1.0> : tensor<16x3x3x8xf32>
+  %cst2 = constant dense<1.0> : tensor<16xf32>
+  %0 = "tfl.pad"(%arg0, %cst0) : (tensor<1x64x64x8xf32>, tensor<4x2xi32>) -> tensor<1x66x66x8xf32>
+  %1 = "tfl.conv_2d"(%0, %cst1, %cst2) {dilation_h_factor = 1 : i32, dilation_w_factor = 1 : i32, fused_activation_function = "NONE", padding = "VALID", stride_h = 1 : i32, stride_w = 1 : i32} : (tensor<1x66x66x8xf32>, tensor<16x3x3x8xf32>, tensor<16xf32>) -> tensor<1x64x64x16xf32>
```

Comment:
Perhaps good to also add a similar test but then for depthwise-conv?

Reply:
Done in [e14ddb2](https://github.com/larq/compute-engine/pull/712/commits/e14ddb254e6f5f0ff44acca5c2013556c033f84f)